vendordeps/NavX.json

```diff
@@ -1,35 +1,36 @@
 {
-    "fileName": "navx_frc.json",
+    "fileName": "NavX.json",
     "name": "KauaiLabs_navX_FRC",
-    "version": "2023.1.0",
+    "version": "2023.0.3",
     "uuid": "cb311d09-36e9-4143-a032-55bb2b94443b",
     "mavenUrls": [
-        "https://raw.githubusercontent.com/rzblue/navx-frc/maven/maven/release/"
+        "https://dev.studica.com/maven/release/2023/"
     ],
-    "jsonUrl": "https://raw.githubusercontent.com/rzblue/navx-frc/maven/navx_frc.json",
+    "jsonUrl": "https://dev.studica.com/releases/2023/NavX.json",
```

Comment:
Did you find a different third party to get NavX code from?

Reply:
I used the json from the navx website https://pdocs.kauailabs.com/navx-mxp/software/roborio-libraries/java/
https://dev.studica.com/releases/2023/NavX.json